### DISCUSSION TITLE
How to change sound level with visual indication

### DISCUSSION THREAD

**Discussion by echo258** [i3/i3#4744] [Q&A]
How would I add a visual indication when I press the volume up or down key? I use PulseAudio.

**Comment by Airblader** [comment]
> a visual indication

There are many ways to visualize something. What did you have in mind? A notification? Your screen flashing white?

> when I press the volume up or down key

Are you sure you want to do it when pressing the volume keys, and not when the volume changes?

**Reply by echo258** [reply]
It would be best to have notification when the volume changes. Like the pop-up in KDE.
